newIDE/app/src/BehaviorsEditor/index.js

```diff
@@ -343,15 +572,36 @@ const BehaviorsEditor = (props: Props) => {
             })}
           </ScrollView>
           <Column>
-            <Line justifyContent="flex-end" expand>
-              <RaisedButton
-                key="add-behavior-line"
-                label={<Trans>Add a behavior</Trans>}
-                primary
-                onClick={() => setNewBehaviorDialogOpen(true)}
-                icon={<Add />}
-                id="add-behavior-button"
-              />
+            <Line noMargin>
+              <LineStackLayout expand>
```

Comment:
You should use 2 ResponseLineStackLayout, here is the outcome on mobile at the moment:
<img width="356" alt="image" src="https://user-images.githubusercontent.com/32449369/230616275-ef28580c-1f4b-464a-8530-7b1de736906f.png">